src/components/Attachments/AttachmentView/AttachmentViewImage/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {memo} from 'react';
+import type {GestureResponderEvent} from 'react-native';
+import type {AttachmentFile} from '@components/Attachments/types';
+import ImageView from '@components/ImageView';
+import PressableWithoutFeedback from '@components/Pressable/PressableWithoutFeedback';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+
+type AttachmentViewImageProps = {
```

Comment:
Pick reusable props from AttachmentViewProps

Reply:
No I meant to use `Pick` e.g. `Pick<AttachmentViewProps, 'file' | 'isAuthTokenRequired'>;`